README.md

```diff
@@ -196,6 +196,22 @@ python3 serve_google_api.py --endpoint localhost:50051 --checkpoint ...
 python3 scripts/stt_google.py --endpoint localhost:50051 --lang ru --api-key-credentials= -i calls_micro/calls_micro.json
 ```
 
+# Frontend performance metrics
+```shell
+bash scripts/read_audio_performance.sh
+```
+| file       | reads count |  backend | process_time ms| perf_counter ms| 
+|------------|:--:|---------:|--------------:|-------------:|
+|test_5s.wav |100 |soundfile |4597.299309    | 800.03763    |
```

Comment:
It should be on the order of 100 microseconds (at least when done without s2f). At least judging from my measurements in https://github.com/vadimkantorov/readaudio/blob/master/decode_audio.py#L189